handsontable/src/plugins/filters/filters.js

```diff
@@ -233,9 +233,13 @@ export class Filters extends BasePlugin {
     }
 
     if (!this.components.get('filter_by_value')) {
+      const pluginSettings = this.hot.getSettings()[PLUGIN_KEY];
+      const uncheckFilteredQueries = pluginSettings?.uncheckFilteredQueries;
```

Comment:
The _Filters_ plugin no longer accepts only `boolean` values. Can I ask you to use `DEFAULT_SETTINGS` and `SETTINGS_VALIDATORS` meta fields in the plugin class? 

Also, you should use an internal method to gather plugin's specific options, e.g., `this.getSetting('uncheckFilteredQueries');`